### PR TITLE
Update pick documentation

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-5409](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5409)
Merge #https://github.com/Ensembl/ensembl-variation/pull/942 and #https://github.com/Ensembl/ensembl-vep/pull/1336 first

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_other.html [hunk 1]
```diff
@@ -256,26 +256,27 @@ <h4 id="pick_options">Options</h4>
     <li style="margin-bottom:20px">
       <b>--pick</b></p>
       <p>This is the option
```
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/script/vep_other.html:258]: Please also delete "This is the option", as it makes no sense without the other half of the sentence.

File: docs/htdocs/info/docs/tools/vep/vep_formats.html [hunk 2]
```diff
@@ -294,6 +294,20 @@ <h2 id="hgvs"> HGVS identifiers</h2>
     <pre class="code sh_sh">LRG_1t1:c.841G>T
 LRG_1:g.10006G>T</pre>
 
+    <h4>Ambiguous gene-based descriptions</h4>
+    <p>It is possible to use ambiguous descriptions listing only gene symbol or UniProt accession and protein change (e.g. PHF21B:p.Tyr124Cys,  P01019:p.Ala268Val), as seen in the literature, though this is not recommended as it can produce multiple different variants at genomic level. The transcripts for a gene are considered in the following order:
+      <ol>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Select transcript status</a></li>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Plus Clinical transcript status</a></li>
```
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/vep_formats.html:301]: Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.

File: docs/htdocs/info/docs/tools/vep/script/vep_other.html [hunk 3]
```diff
@@ -256,26 +256,27 @@ <h4 id="pick_options">Options</h4>
...
-      we anticipate will be of use. VEP chooses one block of
+      VEP chooses one block of
       annotation per variant, using an ordered set of criteria. This order may be
       customised using <a href="vep_options.html#opt_pick_order">--pick_order</a>.
       <ol>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Select transcript status</a></li>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Plus Clinical transcript status</a></li>
         <li>canonical status of transcript</li>
```
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/script/vep_other.html:265]: Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Small tweaks required - can you get them done before the end of this week so we can merge them into the new release?

**Review by nuno-agostinho — APPROVED** [review]
All seems good to me!

File: docs/htdocs/info/docs/tools/vep/vep_formats.html (hunk 2 above)
Comment (ens-ap5) [reply docs/htdocs/info/docs/tools/vep/vep_formats.html:301]: I don't see what's changed here - it should either be "https://www.ensembl.org/info/genome/genebuild/mane.html" or "/info/genome/genebuild/mane.html"

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Not all requested changes seem to have been implemented.

**Comment by nuno-agostinho** [comment]
Hey @ens-ap5, @olaaustine is currently on leave so I updated the PR with the changes you mentioned. I used relative links for consistency with the rest of the file.

The PRs mentioned in the description were already merged, so you can please merge this PR if you find it fine now. Tell me otherwise.

Thanks! 😊

**Review by ens-ap5 — APPROVED** [review]

**Pull request merged**